vhost-device-sound/src/device.rs

```diff
@@ -301,21 +253,19 @@ impl VhostUserSoundThread {
                             p.direction = s.direction as u8;
                             p.channels_min = s.channels_min;
                             p.channels_max = s.channels_max;
-                            buf.extend_from_slice(p.as_slice());
+                            if let Err(err) = writter_content.write_obj(p) {
+                                log::error!("Error: DescriptorWriteFailed: {}", err);
```

Comment:
Is this new log entry intended?

Reply:
I tried to replicate what happened before, e.g., https://github.com/rust-vmm/vhost-device/blob/e19e93055a047d1fb11ad53b149bd11db9abcc8c/vhost-device-sound/src/device.rs#L310, is that OK?

Reply:
That did not do logging itself but just bubbled up the error. I assume the caller of this function then eventually logs it (or bubbles it up further). The new code returns the same error, so if the previous code already logged up in the call stack, this adds a second log before that.